microservices/services/dictionary/service/src/main/frontend/src/functions/formatters.ts

```diff
@@ -16,19 +16,24 @@ export function parseVal(colName: any, colValue: any): string {
 
 // Produces the max substring for the table, adds '...' if above 34 chars.
 export function maxSubstring(str: any, colName: any): any {
-  if (str == undefined) {
+  if (str === undefined) {
     return;
-  } else if ((colName === 'fieldName' || colName === 'internalFieldName') && str.length > 32) {
-    return str.substring(0, 30) + ' ...';
-  } else if ((colName === 'Types') && str.length > 14) {
-    // Types is offset by 2 to prevent overlapping in 'Tokenized' Column
-    return str.substring(0, 9) + ' ...';
-  } else if ((colName === 'Descriptions') && str.length > 24) {
-    return str.substring(0, 22) + ' ...';
-  } else if ((colName === 'CopyPaste') && str.length > 42) {
-    return str.substring(0, 40) + ' ...';
-  } else {
-    return str;
+  }
+
+  switch (colName) {
+    case 'fieldName':
+    case 'internalFieldName':
+      return str.length > 32 ? str.substring(0, 30) + ' ...' : str;
+    case 'Types':
+      return str.length > 14 ? str.substring(0, 9) + ' ...' : str;
```

Comment:
Can you add back the comment here?
```suggestion
      // Types is offset by 2 to prevent overlapping in 'Tokenized' Column
      return str.length > 14 ? str.substring(0, 9) + ' ...' : str;
```